### DISCUSSION TITLE
Add a run-coalton binary

### DISCUSSION THREAD

**Discussion by stylewarning** [coalton-lang/coalton#977] [Ideas and Proposals]
It would be nice if new users could immediately type and run Coalton without the rigamarole. Making a binary that can read files, load them, and run an entry point would be useful.

**Comment by macrologist** [comment]
Something like 

`./coalton --load mine.lisp --load yours.lisp --start mine::start --args '(1"two" 3.0)' `   ??

 Could it wrap a REPL too that reads expressions as if they appear in a `(coalton ...)` ?
 
 `./coalton --repl` 
 
Also: Were you considering a binary with a help menu and documented options? Have you used `clon` or `clingon`? Of the two I prefer `clingon`.

**Reply by stylewarning** [reply]
Something like what you're saying, yes, on all points. We would need to experiment to see what's useful (e.g., auto-COALTON-TOPLEVEL) of course.
